MANIFEST.md

```diff
@@ -0,0 +1,16 @@
+Manifest 
+---------
+
+Celem naszych spotkań jest poszerzanie wiedzy i wspólne spędzanie czasu w przyjaznej atmosferze. W związku z tym oczekujemy, że wszyscy bez wyjątku będą darzyć się wzajemnym szacunkiem.
+
+Jako społeczność oczekujemy, by każdy z nas stosował się do przedstawionych poniżej zasad, ponieważ uważamy, że pozwoli to wszystkim komfortowo i bezstresowo uczestniczyć w życiu naszej grupy:
+
+* wroc.py jako miejsce przyjazne dla wszystkich pozostaje przestrzenią wolną od formułowania publicznych ocen w kwestiach światopoglądowych, zwłaszcza tych związanych z płcią, preferencjami seksualnymi, budową ciała, rasą lub wyznaniem.
```

Comment:
Jeżeli porównujemy spotkanie wroc.py do koncertu to chciałbym się dowiedzieć kto organizuje spotkania wroc.py. Jaką strukturę wasza organizacja posiada? Czy posiadacie adres korespondencyjny? Czy można założyć, że sankcje za wykroczenia od regulaminu są również "zwyczajowe" czy zamierzacie je dokładnie ustalić? Czy jestem członkiem waszej organizacji? Czy mogę się stać jej członkiem? Czy skoro nie jestem członkiem waszej organizacji to czy mogą mnie dotyczyć zapisy waszego regulaminu? Czy muszę podpisać ten regulamin?